### ISSUE TITLE
Setting `opened` property opens dropdown-menu and then closes it

### ISSUE THREAD

**Issue by samiheikki** [vaadin/vaadin-select#167]
### Description
Look at the live demo. I have `document.querySelector('vaadin-dropdown-menu').opened = true
` at the end of js-file and dropdown opens but then it closes again (you might have to press `Run` couple of times to see it).

#### Live Demo
https://jsfiddle.net/samiheikki/usmcyrxg/2/

#### Expected Results
Dropdown remains open

#### Actual Results
Dropdown doesn't remain open

**Comment by murtukov** [comment]
Exactly the same problem by me

**Comment by tomivirkki** [comment]
Changing the version in the fiddle from `1.2.0-alpha2` to `1.2.0-beta1` fixes it for me

**Comment by limonte** [comment]
Fixed for `vaadin-dropdown-menu` in `1.2.0-beta1`.

Also, `vaadin-dropdown-menu` was renamed to `vaadin-select` and the same issue is fixed in `vaadin-select@2.0.0-beta1`.

Proof: https://interesting-surf.glitch.me/

**Comment by murtukov** [comment]
Actually his problem was that he forgot to put select options in <vaadin-dropdown-menu>
```
<vaadin-dropdown-menu>
  <!-- here -->
</vaadin-dropdown-menu>
```

So, @samiheikki your example on jsfiddle will work if you change your code to:
```
<vaadin-dropdown-menu>
  <template>
    <vaadin-list-box>
      <vaadin-item>Jose</vaadin-item>
      <vaadin-item>Manolo</vaadin-item>
      <vaadin-item>Pedro</vaadin-item>
    </vaadin-list-box>
  </template>
</vaadin-dropdown-menu>
```

And in my case I used React, and the problem was the fact, that JSX can't handle **`<tempalte>`** tag properly, so you need to create a helper component and use this component instead of **`<tempalte>`** tag, like follows:

```
// Create helper React component
function Template({ children, ...attrs }) {
    return (
        <template
            {...attrs}
            dangerouslySetInnerHTML={{ __html: children }}
        />
    );
}

// Use this component instead of template tag
<vaadin-dropdown-menu>
  <Template>
    {`
      <vaadin-list-box>
        <vaadin-item>Jose</vaadin-item>
        <vaadin-item>Manolo</vaadin-item>
        <vaadin-item>Pedro</vaadin-item>
      </vaadin-list-box>
    `}
  </Template>
</vaadin-dropdown-menu>
```

Don't forget to use `braces` and `backquotes`
